src/view-compiler.js

```diff
@@ -306,6 +306,9 @@ export class ViewCompiler {
       }
       return node.nextSibling;
     } else if (tagName === 'template') {
+      if(!node.content){
```

Comment:
On MDN they test using `if ('content' in document.createElement('template')) {`, so maybe it would be preferred to have the condition be `if (!('content' in node)) {`
Then it wouldn't begin to check the value of `content`